dataland-frontend/src/components/forms/parts/fields/SingleSelectFormField.vue

```diff
@@ -1,25 +1,58 @@
 <template>
-  <div class="form-field">
+  <div :class="containerClass">
     <UploadFormHeader :label="label" :description="description" :is-required="required" />
     <SingleSelectFormElement
       :name="name"
       :validation="validation"
       :validation-label="validationLabel ?? label"
       :placeholder="placeholder"
       :options="options"
+      :input-class="inputClass"
+      :is-required="required"
```

Comment:
manchmal isRequired, manchmal :required="true", manchmal is-required="true", funktioniert das alles gleich?
dann würde ich das einheitlich machen.